Makefile.third

```diff
@@ -204,6 +204,7 @@ $(CRENGINE_LIB): $(ZLIB) $(ZSTD_LIB) $(PNG_LIB) $(FREETYPE_LIB) $(HARFBUZZ_LIB)
 		-DUTF8PROC_INCLUDE_DIR="$(UTF8PROC_DIR)" \
 		-DZSTD_INCLUDE_DIR="$(ZSTD_DESTDIR)/include" \
 		-DCMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE) \
+		-DSRELL_INCLUDE_DIR="${SRELL_INCLUDE_DIR}" \
```

Comment:
Move it one line above, with the other *_INCLUDE_DIR.
-DCMAKE_BUILD_TYPE looks different :)